core/controllers/skill_editor.py

```diff
@@ -90,6 +106,18 @@ def check_can_edit_skill_description(user):
 class SkillRightsHandler(base.BaseHandler):
     """A handler for returning skill rights."""
 
+    URL_PATH_ARGS_SCHEMAS = {
+        'skill_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
```

Comment:
This should be the same rule as for the handler above.

Reply:
@vojtechjelinek I have done the changes you have requested,
Brief:
-added regex validation for skillRightsHandler class
-removed HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS from payload_validator.py
-removed skillEditorPage and skillRightsHandler from HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS inside handler_schema_constants.py